tests/spark/autologging/ml/test_pyspark_ml_autologging.py

```diff
@@ -485,7 +485,7 @@ def test_param_search_estimator(  # pylint: disable=unused-argument
         assert math.isclose(avg_metric_value, run_data.metrics[avg_metric_name], rel_tol=1e-6)
         assert math.isclose(avg_metric_value, float(row.get(avg_metric_name)), rel_tol=1e-6)
 
-        if isinstance(estimator, CrossValidator) and Version(pyspark.__version__) > Version("3.2"):
+        if isinstance(estimator, CrossValidator) and Version(pyspark.__version__) >= Version("3.3"):
```

Comment:
The `CrossValidationModel.stdMetrics` was added in spark 3.3,
now spark 3.2.1 come out it will break the test.